Gemfile

```diff
@@ -11,9 +11,10 @@ group :development do
   gem "juwelier", "~> 2.1.0"
 end
 
-gem 'sinatra'
-gem "git"
-gem "diffy"
 gem "actionview"
+gem "ansispan"
+gem "diffy"
+gem "git"
+gem 'sinatra'
```

Comment:
I can't find anything to nitpick, so I'll offer this: perhaps `rubocop -a` might help with style consistency? E.g. here there's a lonely single-quoted string.